pkg/pgxconn/pgx_conn.go

```diff
@@ -37,21 +47,63 @@ type connImpl struct {
 	Conn *pgxpool.Pool
 }
 
+type pgxRows struct {
+	pgx.Rows
+	sqlQuery  string
+	args      []interface{}
+	startTime time.Time
+	loggged   bool
+}
+
 func (p *connImpl) Close() {
 	conn := p.Conn
 	p.Conn = nil
 	conn.Close()
 }
 
-func (p *connImpl) Exec(ctx context.Context, sql string, arguments ...interface{}) (pgconn.CommandTag, error) {
-	return p.Conn.Exec(ctx, sql, arguments...)
+func (p *pgxRows) Next() bool {
+	if !p.loggged {
+		p.loggged = true
```

Comment:
```suggestion
	if !p.logged {
		p.logged = true
```
I think it should be `hasLogged`.